### PR TITLE
AWS OIDC account 

### PR DESCRIPTION
**Pull request by benPearce1**

### CODE REVIEW

File: src/pages/docs/projects/variables/aws-account-variables.md [hunk 1]
```diff
@@ -42,9 +53,24 @@ Each of the above properties can be referenced in PowerShell.
 Write-Host 'AwsAccount.Id=' $OctopusParameters["aws account"]
 Write-Host 'AwsAccount.AccessKey=' $OctopusParameters["aws account.AccessKey"]
 
+# For an OpenId Connect account
+Write-Host `AwsAccount.RoleArn=` $OctopusParameters["aws account.RoleArn"]
+Write-Host `AwsAccount.SessionDuration=` $OctopusParameters["aws account.SessionDuration"]
```
Comment (IsaacCalligeros95) [inline src/pages/docs/projects/variables/aws-account-variables.md:58]: Do we want to add the JWT example here?

File: src/pages/docs/projects/variables/aws-account-variables.md [hunk 2]
```diff
@@ -42,9 +53,24 @@ Each of the above properties can be referenced in PowerShell.
...
+
 # Directly as a variable
 Write-Host 'AwsAccount.Id=' #{aws account}
 Write-Host 'AwsAccount.AccessKey=' #{aws account.AccessKey}
+
+# For an OpenId Connect account
+Write-Host `AwsAccount.RoleArn=` #{aws account.RoleArn}
+Write-Host `AwsAccount.SessionDuration=` #{aws account.SesstionDuration}
```
Comment (IsaacCalligeros95) [inline src/pages/docs/projects/variables/aws-account-variables.md:66]: And here?

File: src/pages/docs/projects/variables/aws-account-variables.md (hunk 1 above)
Comment (benPearce1) [reply src/pages/docs/projects/variables/aws-account-variables.md:58]: the jwt is sensitive, so it won't write anything useful anyway. I noticed that the other example also leaves out the secret too

**Review by IsaacCalligeros95 — COMMENTED** [review]
Content looks good to me, I'm not sure if we want to add writing out the JWT in the examples, It's a sensitive value so is likely redundant to add. I'll leave approving the updates to Steve

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM

Just check that you're happy with my spelling fixes here and then merge away :)

https://github.com/OctopusDeploy/docs/pull/2136/commits/4ba6ba3f7b885184c996b4726d30458aef77574c

**Pull request merged**